paths/intro-to-github/00-welcome.md

```diff
@@ -1,66 +1,54 @@
 ---
-layout: intro-to-github
+layout: simple-class
 header:
   overlay_image: cover.jpeg
   overlay_filter: rgba(46, 129, 200, 0.6)
 title: Welcome
 permalink: /intro-to-github/
-facilitator: false
+next-page: /intro-to-github/01/
 sidebar:
   nav: "intro-nav"
----
-
-## Welcome
-
-If you are looking for a quick and fun introduction to GitHub, you've found it. This class is designed to get you started using GitHub in less than an hour.
-
-## What is GitHub?
-
-We are glad you asked! Many people stumble onto GitHub through one of the millions of Open Source projects it holds or because their employer or professor is using it. Why do these projects use GitHub?
+main-content: |
+  ## We're Glad You're Here :octocat:
 
-**At its heart, GitHub is a collaboration platform.**
+  This course is designed to get you started using GitHub in less than an hour.
 
-People around the world use GitHub to collaborate on projects ranging from software to policy documents and cookbooks. You can share your projects with the world and invite your friends to help, or you can keep your projects private and still have easy access wherever you are.
+  **In this course, you will learn how to**:
 
-**GitHub is also a powerful version control tool.**
+  - Use the GitHub.com website to complete the GitHub workflow
+  - Collaborate with ~~strangers~~ new friends on a fun project
```

Comment:
@hollenberry This isn't striking out for me either. A quick search shows [we may need to alter our configs for Jekyll slightly](http://stackoverflow.com/questions/17004095/jekyll-markdown-strikethrough).